assets/js/plan_template.js

```diff
@@ -140,7 +140,7 @@ async function postPlanTemplate() {
     console.log(`data about to send: ${JSON.stringify(data)}`);
 
     axios.post(
-        url, JSON.stringify(data), { timeout: 10000 }
+        url, JSON.stringify(data), { timeout: 15000 }
```

Comment:
why increase ?

Reply:
Good question, this makes sure the front end requests do not timeout before backend computations time outs happen.